### PR TITLE
fix link, add space

### PR DESCRIPTION
**Pull request by quiddity-wp**
the link was opening the widgets page

### CODE REVIEW

File: visual-style_icons.html [hunk 1]
```diff
@@ -109,15 +109,15 @@ <h3>Usage</h3>
...
-					<p>The <a href="https://doc.wikimedia.org/oojs-ui/master/demos/#icons-mediawiki-ltr">OOUI icon repository</a> provides a list of the currently available icons.</p>
+					<p>The <a href="https://doc.wikimedia.org/oojs-ui/master/demos/?page=icons&theme=wikimediaui&direction=ltr&platform=desktop#icons-mediawiki-ltr">OOUI icon repository</a> provides a list of the currently available icons.</p>
 
 					<h3 id="References">References</h3>
 					<ol>
 						<li>Thanks to volunteer contributor <a href="https://phabricator.wikimedia.org/p/SamanthaNguyen/" target="_blank">@SamanthaNguyen</a> for many suggestions on the “Principles” section in icons on <a href="https://phabricator.wikimedia.org/T155684" target="_blank">Phabricator task T155684</a>.</li>
-						<li>1&#160;dp equals 1&#160;px in CSS at 100% zoom level and 1x device.<a href="https://en.wikipedia.org/wiki/Device-independent_pixel" target="_blank">Device-independent pixel on English Wikipedia</a>.</li>
+						<li>1&#160;dp equals 1&#160;px in CSS at 100% zoom level and 1x device. <a href="https://en.wikipedia.org/wiki/Device-independent_pixel" target="_blank">Device-independent pixel</a> on English Wikipedia.</li>
```
Comment (Volker-E) [inline visual-style_icons.html]: Wondering about putting forward the end tag?

**Comment by Volker-E** [comment]
Mostly, changes look great. Thanks for the contributions @quiddity-wp! Only one small request…

File: visual-style_icons.html (hunk 1 above)
Comment (quiddity-wp) [reply visual-style_icons.html]: Oh, feel free to drop that part. -- After I added the missing space before the link, I changed the link to follow Wikipedia's own standard link formatting, but that's possibly (?) not ideal here. 
I was also contemplating changing the line to: "See [Device-independent pixel] on English Wikipedia." as an alternative. 
Do as thy will. :)

**Comment by Volker-E** [comment]
Thanks @quiddity-wp!

**Pull request merged**